ResearchStack/ResearchStack
# Bug/pat 819 completion step middle flow

A completion step in the middle of a task should terminate the task unless it is the trigger of a branching scenario.

### To test:
_Check out the following Axon SDK MR:_ 
https://gitlab.medable.com/axon/android/sdk/merge_requests/425

- int-dev | hybridstudy | Julien
- julien_bp+1@medable.com | jjulienn

**Scenario 1:** Completion in middle of ordered task (no branching)

- Log into the study, select the task: **Completion second place nobranch**
- Follow steps
- Verify that the task is terminated after Completion task, even though there is another step configured in that task.

**Scenario 2:** Completion in the middle of a branching scenario, and the completion step is the trigger

- Log into the study, select the task: ** Completion branchign **
- Follow steps
- Verify that you are taken to the next step after Competion step, as it is configured as the trigger for the branching.

**Scenario 3:** Completion in the middle of a branching scenario, but is not the trigger

- Log into the study, select the task: ** Branching task but no completion **
- Follow steps
- Verify that the task is terminated after Completion task, even though there is another step configured in that task.
